dev-packages/scripts/che-theia.ts

```diff
@@ -28,11 +28,11 @@ function run(script) {
     });
 }
 
-function getCheScript() {
-    const commandIndex = process.argv.findIndex(arg => arg.endsWith('che:script')) + 1;
+function getCheScript(): string {
```

Comment:
Maybe rename here to: `getCheTheiaScript` What do you think?

Reply:
But up to you.

Reply:
OK